### ISSUE TITLE
Can't define some values that equals false with Boolean operation

### ISSUE THREAD

**Issue by tz5514** [FormidableLabs/babel-plugin-transform-define#37]
There's a bug that we can't defined some "false value", like false, 0, ''(empty string), undefined, null
just like following code
It will be ignored in babel compiling. 

```
{
  "plugins": [
    ["transform-define", {
      "EMPTY_STRING": "",
      "FALSE_BOOLEAN": false,
      "ZERO_NUMBER": 0,
      "NULL": null
    }]
  ]
}
```

**Comment by bspaulding** [comment]
Looks like this is fixed on master, but not released to npm?

**Comment by baer** [comment]
This is fixed in 1.3! Hoooray!

**Comment by tz5514** [comment]
thanks for fixing!
